recipes/maven/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "3.6.3":
+    url: https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
```

Comment:
I took a peak the source code is available...

https://downloads.apache.org/maven/maven-3/3.6.3/source/apache-maven-3.6.3-src.tar.gz

Im curious what would be the requirements for building this

Reply:
I also did not try to build from source, but I guess that it is quite large, e.g. the following site warns that the full source build builds 400 modules... And then the question is, which modules should we build here?

https://maven.apache.org/scm.html

Reply:
Are all of those modules in this binary download?

Reply:
That's a good question which I cannot answer for sure... I couldn't really find out which modules are included in the bin download.
My guess is that the bin download only contains the necessary basis.